plugin/completion.py

```diff
@@ -71,8 +71,7 @@ def show_popup(self, minihtml_content: str) -> None:
             self.view,
             minihtml_content,
             flags=sublime.COOPERATE_WITH_AUTO_COMPLETE,
-            max_width=480,
-            max_height=410,
+            max_width=800,
```

Comment:
Any specific reason why width was restricted?

Reply:
We could try:

- `view.em_width() * 100` (does this mean approximately hundred characters per line?)
- `view.layout_extent()[0]` (This seems to be viewport minus sidebar?)
- `view.viewport_extent()[0]` (This seems to be the entire window)

Reply:
What do we want to achieve with that?
I believe popup is already restricted to viewport size by default.

Reply:
If you use a 4K screen and your viewport width is 3840 pixels wide, then 800 pixels is tiny.

Reply:
If I don't specify `max_width` then it breaks very prematurely:
![Screenshot 2020-05-13 at 22 12 22](https://user-images.githubusercontent.com/153197/81860288-d6225b00-9566-11ea-8048-d14198d62665.png)

`em_width() * 100`:
![Screenshot 2020-05-13 at 22 11 52](https://user-images.githubusercontent.com/153197/81860232-c440b800-9566-11ea-9d3e-08b490b5757e.png)

`view.viewport_extent()[0]`:
![Screenshot 2020-05-13 at 22 11 07](https://user-images.githubusercontent.com/153197/81860172-a96e4380-9566-11ea-8128-039d3f39f391.png)

Reply:
I'll let you be the judge. I will say that 800 is not future-proof :)

Reply:
I'd probably choose some ridiculously high value since ST will clamp to viewport size anyway but there might be gotchas with it.
Let's go for viewport width for now.

Reply:
Yeah I tried `sys.maxsize` but that made the popup appear at the location of the AC for some reason.